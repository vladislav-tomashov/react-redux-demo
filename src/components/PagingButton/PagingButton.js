import React from "react";
import "./PagingButton.scss";

const PagingButton = props => <button className="PagingButton" {...props} />;

export { PagingButton as default };
