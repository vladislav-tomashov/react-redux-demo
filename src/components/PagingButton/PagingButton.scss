@import "../../styles/settings";

.PagingButton {
  background: $blue;
  border: 2px solid $light-blue;
  color: white;
  display: inline-block;
  font-size: $font-size-large;
  font-weight: 300;
  line-height: 1;
  padding: $s-size;
  text-decoration: none;
  min-width: 8rem;
  margin: $xs-size;

  &:disabled {
    color: darken(white, 50%);
    background: darken($blue, 20%);
    border: 2px solid darken($light-blue, 50%);
  }
}
