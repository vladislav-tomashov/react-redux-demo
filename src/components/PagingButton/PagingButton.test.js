import React from "react";
import { shallow } from "enzyme";
import PagingButton from "./PagingButton";

describe("PagingButton component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<PagingButton>Test</PagingButton>);
  });

  test("should render PagingButton correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
