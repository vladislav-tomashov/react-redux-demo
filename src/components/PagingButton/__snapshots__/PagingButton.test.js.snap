// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagingButton component should render PagingButton correctly 1`] = `undefined`;
